### PR TITLE
prepare tools release announcment

### PR DESCRIPTION
**Pull request by mashehu**

### CODE REVIEW

**Review by mirpedrol — APPROVED** [review]

File: src/content/blog/2024/tools-2_12.mdx [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+title: 'nf-core/tools - 2.12.0'
+subtitle: 'TUI or not TUI 💀'
```
[the remark this answers is not in the document]
Comment (mashehu) [reply src/content/blog/2024/tools-2_12.mdx:3]: not shure if people get the pun, but let's see 🙂

File: src/content/blog/2024/tools-2_12.mdx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+-   Fun changes on the tooling side:
+    -   The longer CI tests for the tools repo are now run on self-hosted runners on
+        AWS (thanks for the sponsorship AWS!).
+    -   We now use [ruff](https://github.com/astral-sh/ruff) for linting and formatting, goodbye to Black, isort and pyupgrae
+        and thank you for your service!🫡.
+        :::tip
+        -   We included ruff to the pre-commit config. Use `pre-commit install` to install the git hook scripts.
+        -   To lint on save with VSCode, add the following settings:
+        ```
+        foo
```
[the remark this answers is not in the document]
Comment (mashehu) [reply src/content/blog/2024/tools-2_12.mdx:54]: oops sorry.

File: src/content/blog/2024/tools-2_12.mdx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    -   👀 fixing action has started
+    -   👍🏻 Everything looks good, nothing to fix
+    -   🎉 Fixed errors and commited the changes
+    -   😕 Something went wrong, please check the logs (also a comment with a link
+        to the logs is added)
+        <Image src={fix_linting} alt="Screenshot of the nf-core-bot fix linting comment" />
+-   The `nf-core tui` subcommand launches a TUI (terminal user interface) to
+    intuitively explore the command line flags, built using
+    [Trogon](https://github.com/Textualize/trogon) (more TUIs to come!)
+    <img src={tui.src} alt="Demo gif of the TUI." class="img-fluid" />.
```
Comment (mirpedrol) [inline src/content/blog/2024/tools-2_12.mdx:37]:
```suggestion
    <img src={tui.src} alt="Demo gif of the TUI." class="img-fluid" />
```

File: src/content/blog/2024/tools-2_12.mdx [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+            "editor.defaultFormatter": "charliermarsh.ruff"
+        }
+        ```
+
+        -   To run ruff manually you can use `ruff check .` and `ruff format .`.
+
+        :::
+
+-   We've got a new bot which helps us to keep the changelog up to date.
+    Big thanks to <Profile username={"vladsavelyev"} size={25}/> for the code!
```
Comment (mirpedrol) [inline src/content/blog/2024/tools-2_12.mdx:73]: The profile looks cut, at least on my screen
image

Comment (mashehu) [reply src/content/blog/2024/tools-2_12.mdx:73]: yeah, wasting time on that at the moment... will just use a normal link here

**Pull request merged**